cmd/orb-server/startcmd/start.go

```diff
@@ -338,35 +338,34 @@ func startOrbServices(parameters *orbParameters) error {
 
 	defer monitoringSvc.Close()
 
+	proofHander := proof.New(
+		&proof.Providers{
+			Store:         vcStore,
+			MonitoringSvc: monitoringSvc,
+			DocLoader:     orbDocumentLoader,
+		},
+		vcCh)
+
 	activityPubService, err := apservice.New(apConfig,
 		apStore, t, sigVerifier,
 		// TODO: Define all of the ActivityPub handlers
 		// TODO: mockProofHandler need to be replaced with real implementation which should invoke Watch function.
-		apspi.WithProofHandler(mockProofHandler(func(anchorCredID string, _, endTime time.Time, proof []byte) error {
-			return monitoringSvc.Watch(anchorCredID, endTime, proof)
-		})),
+		apspi.WithProofHandler(proofHander),
```

Comment:
`mockProofHandler` needs to be removed and this todo `TODO: mockProofHandler need to be replaced with real implementation which should invoke Watch function.`